src/VisualStudio/Core/Def/Implementation/AbstractVsTextViewFilter`2.cs

```diff
@@ -32,7 +33,20 @@ internal abstract class AbstractVsTextViewFilter<TPackage, TLanguageService> : A
             _languageService = languageService;
         }
 
-        public virtual int GetDataTipText(TextSpan[] pSpan, out string pbstrText)
+        int IVsTextViewFilter.GetDataTipText(TextSpan[] pSpan, out string pbstrText)
```

Comment:
No, this is explicit interface method implementation. No modifiers are applicable.